modules/ROOT/pages/aggregator/aggregator-examples.adoc

```diff
@@ -74,9 +74,9 @@ To make sure that the logic that processes all aggregated elements executes once
 ----
 
 For this case, the data types that come from the `aggregation-complete` route and `aggregator-listener` are the same. So, for the execution of the flow `whatToDoWithAllTheseNames`, it does not matter where the data comes from.
-However, with this configuration, in the case the aggregation is complete, the flow `whatToDoWithAllTheseNames` will trigger twice, once from the `aggregation-complete` route `flow-ref` and again from the `flow-ref` in the flow with the `aggregator-listener`.
+However, with this configuration, in the case the aggregation is complete, the flow `whatToDoWithAllTheseNames` will trigger twice: once from the `aggregation-complete` route `flow-ref`, and again from the `flow-ref` in the flow with the `aggregator-listener`.
 
-To avoid multiple executions of the same logic, the `flow-ref` after the `aggregator-listener` should be wrapped in a `choice` component that uses the `isAggregationComplete` attribute to check for an incomplete aggregation. That way, the `flow-ref` is only called when a timeout triggered that flow.
+To avoid multiple executions of the same logic, the `flow-ref` after the `aggregator-listener` should be wrapped in a `choice` component that uses the `isAggregationComplete` attribute to check for an incomplete aggregation. That way, the `flow-ref` is called only when a timeout triggered that flow.
```

Comment:
Again suggest splitting para text around the example:
```suggestion
To avoid multiple executions of the same logic, wrap the `flow-ref` value `aggregator-listener` in a `choice` component that uses the `isAggregationComplete` attribute to check for an incomplete aggregation:

Using this syntax, `flow-ref` is called only when a timeout triggers that flow.
```